packages/color-scale-block/src/helpers/getMinimumAmountOfSquares.ts

```diff
@@ -0,0 +1,11 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Settings } from '../types';
+import { MAXIMUM_NUMBER_OF_PLACEHOLDER_SQUARES, MINIMUM_NUMBER_OF_COLORS } from './constants';
+
+export const getMinimumAmountOfSquares = (blockSettings: Settings) => {
+    const addedFirstColor = blockSettings.colorInput
+        ? blockSettings.colorInput.filter((item) => item).length > 0
+        : false;
+    return addedFirstColor ? MINIMUM_NUMBER_OF_COLORS : MAXIMUM_NUMBER_OF_PLACEHOLDER_SQUARES;
```

Comment:
```suggestion
        : false;

    return addedFirstColor ? MINIMUM_NUMBER_OF_COLORS : MAXIMUM_NUMBER_OF_PLACEHOLDER_SQUARES;
```

Let make the code breath a bit :)